Xamarin.Forms.Core/ColorTypeConverter.cs

```diff
@@ -6,171 +6,253 @@ namespace Xamarin.Forms
-					case "Magenta": return Color.Magenta;
-					case "Maroon": return Color.Maroon;
-					case "MediumAquamarine": return Color.MediumAquamarine;
-					case "MediumBlue": return Color.MediumBlue;
-					case "MediumOrchid": return Color.MediumOrchid;
-					case "MediumPurple": return Color.MediumPurple;
-					case "MediumSeaGreen": return Color.MediumSeaGreen;
-					case "MediumSlateBlue": return Color.MediumSlateBlue;
-					case "MediumSpringGreen": return Color.MediumSpringGreen;
-					case "MediumTurquoise": return Color.MediumTurquoise;
-					case "MediumVioletRed": return Color.MediumVioletRed;
-					case "MidnightBlue": return Color.MidnightBlue;
-					case "MintCream": return Color.MintCream;
-					case "MistyRose": return Color.MistyRose;
-					case "Moccasin": return Color.Moccasin;
-					case "NavajoWhite": return Color.NavajoWhite;
-					case "Navy": return Color.Navy;
-					case "OldLace": return Color.OldLace;
-					case "Olive": return Color.Olive;
-					case "OliveDrab": return Color.OliveDrab;
-					case "Orange": return Color.Orange;
-					case "OrangeRed": return Color.OrangeRed;
-					case "Orchid": return Color.Orchid;
-					case "PaleGoldenrod": return Color.PaleGoldenrod;
-					case "PaleGreen": return Color.PaleGreen;
-					case "PaleTurquoise": return Color.PaleTurquoise;
-					case "PaleVioletRed": return Color.PaleVioletRed;
-					case "PapayaWhip": return Color.PapayaWhip;
-					case "PeachPuff": return Color.PeachPuff;
-					case "Peru": return Color.Peru;
-					case "Pink": return Color.Pink;
-					case "Plum": return Color.Plum;
-					case "PowderBlue": return Color.PowderBlue;
-					case "Purple": return Color.Purple;
-					case "Red": return Color.Red;
-					case "RosyBrown": return Color.RosyBrown;
-					case "RoyalBlue": return Color.RoyalBlue;
-					case "SaddleBrown": return Color.SaddleBrown;
-					case "Salmon": return Color.Salmon;
-					case "SandyBrown": return Color.SandyBrown;
-					case "SeaGreen": return Color.SeaGreen;
-					case "SeaShell": return Color.SeaShell;
-					case "Sienna": return Color.Sienna;
-					case "Silver": return Color.Silver;
-					case "SkyBlue": return Color.SkyBlue;
-					case "SlateBlue": return Color.SlateBlue;
-					case "SlateGray": return Color.SlateGray;
-					case "Snow": return Color.Snow;
-					case "SpringGreen": return Color.SpringGreen;
-					case "SteelBlue": return Color.SteelBlue;
-					case "Tan": return Color.Tan;
-					case "Teal": return Color.Teal;
-					case "Thistle": return Color.Thistle;
-					case "Tomato": return Color.Tomato;
-					case "Transparent": return Color.Transparent;
-					case "Turquoise": return Color.Turquoise;
-					case "Violet": return Color.Violet;
-					case "Wheat": return Color.Wheat;
-					case "White": return Color.White;
-					case "WhiteSmoke": return Color.WhiteSmoke;
-					case "Yellow": return Color.Yellow;
-					case "YellowGreen": return Color.YellowGreen;
+					switch (color.ToLowerInvariant()) {
```

Comment:
@samhouts I bet it was fun to review, trying to find the one place an uppercase was left, or the one time I mixed `gray` with `blue`, accidentally creating the `glue` color.